pysgrid/sgrid.py

```diff
@@ -208,20 +220,15 @@ def save_as_netcdf(self, filepath):
                                                      center_lat_obj.dtype, 
                                                      center_lat_obj.dimensions
                                                      )
-            grid_node_lon = nclocal.createVariable(node_lon, 
-                                                   node_lon_obj.dtype, 
-                                                   node_lon_obj.dimensions
-                                                   )
-            grid_node_lat = nclocal.createVariable(node_lat, 
-                                                   node_lat_obj.dtype, 
-                                                   node_lat_obj.dimensions
-                                                   )
             grid_var_obj = getattr(self, grid_var)
             grid_vars = nclocal.createVariable(grid_var, grid_var_obj.dtype)
             # not the most robust here... time and angle are hard-coded
             # need to address this
-            time_obj = getattr(self, 'time')
-            grid_time = nclocal.createVariable('time', 
+            try:
+                time_obj = getattr(self, 'time')
+            except AttributeError:
+                time_obj = getattr(self, 'Times')
+            grid_time = nclocal.createVariable(time_obj.variable, 
```

Comment:
Are `time` and `Times` the only expected values to search?  If this list grows I recommend the following approach

```python
search_attr_list = ['time', 'Time']

filter(None, [getattr(a, attr, None) for attr in attr_list])
```